screens/Login.tsx

```diff
@@ -0,0 +1,85 @@
+import { StyleSheet, TextInput, Text, View, Button } from 'react-native';
+import Form from 'react-native-form';
+import { useState } from 'react';
+
+export const Login = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const submitForm = () => {
+    // query firebase
+  };
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.h1_heading}>
+        Welcome back!
+        <br />
+        Please login.
+      </Text>
+
+      <Form style={styles.form_container}>
+        <View>
+          <Text style={styles.h4_heading}>Email</Text>
+
+          <View>
+            <TextInput
+              onChangeText={newText => setEmail(newText)}
+              style={styles.form_field}
+              value={email}
+            />
+          </View>
+        </View>
+
+        <View>
+          <Text style={styles.h4_heading}>Password</Text>
```

Comment:
Make sure you rebase onto `main` -- Ethan's PR #3 introduced some color/font styles that might be relevant here.